CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.5.12
```

Comment:
This is probably 6.6.0 as this is adding a new feature.

Reply:
Thanks, I was in two minds about if this counted as a feature - but agree it is!